### ISSUE TITLE
DNSMasq crashes and never gets up again

### ISSUE THREAD

**Issue by nicopace** [libremesh/lime-packages#111]
# Expected behaviour

After a DNSMasq crash, procd should get dnsmasq up again.

# Actual behaviour

@nicoechaniz saw that it is not getting up again.

I guess @altergui has been working on it.

One proposed workaround could be to replace dnsmasq with udhcpd and a tiny dns replacement. There are some replacements in https://wiki.openwrt.org/doc/howto/dns.overview

**Comment by p4u** [comment]
http://lists.infradead.org/pipermail/lede-dev/2017-March/006828.html

**Comment by ilario** [comment]
This should be fixed by Smonit merge in #142 and #177

**Comment by nicopace** [comment]
@ilario have you checked this?

**Comment by ilario** [comment]
No, actually my assumption was wrong. Smonit up to now just checks if dnsmasq is running as is commented in #142 by @G10h4ck.
Do someone have experience with this? How can this be checked by smonit?

**Comment by nicopace** [comment]
@ilario I believe would be healthier if instead of closing an inactive issue, ask for feedback or test it. That way we prevent loosing issues in the issue bushes :)
